src/params/grid_constants_cbl.F90

```diff
@@ -45,12 +46,17 @@ MODULE grid_constants_mod_cbl
 INTEGER, PARAMETER :: nsCs      = 2  ! # soil carbon stores
 INTEGER, PARAMETER :: nvCs      = 3  ! # vegetation carbon stores
 INTEGER, PARAMETER :: ICE_SoilType = 9 ! SoilType Index (soilparm_cable.nml JAC)
-INTEGER, PARAMETER :: lakes_cable  = 16! SoilType Index (soilparm_cable.nml JAC)
-
-INTEGER, PARAMETER :: ICE_VegType = 17  ! permanent ice index for veg
-
-INTEGER, PARAMETER :: mf = 2          ! # leaves (sunlit, shaded)
-INTEGER, PARAMETER :: niter = 4       ! number of iterations for za/L
+#ifndef UM_CBL
+! UM_CBL gets from cable_surface_types namelist
+INTEGER, PARAMETER :: lakes_cable  = 16 ! cable_surface_type (nml) Index 
+INTEGER, PARAMETER :: ice_cable    = 17 ! cable_surface_type (nml) Index
+#endif
+INTEGER, PARAMETER :: mf          = 2  ! # leaves (sunlit, shaded)
+INTEGER, PARAMETER :: niter       = 4  ! number of iterations for za/L
+INTEGER, PARAMETER :: swb         = 2  ! # SW bands (VIS+NIR) - CM3 alloc TYPEs
+INTEGER, PARAMETER :: nCpool_casa = 10 
+INTEGER, PARAMETER :: nNpool_casa = 10 
+INTEGER, PARAMETER :: nPPool_casa = 12 
```

Comment:
@JhanSrbinovsky These are not used anywhere! Please remove. Or prove me wrong 😄 
AM3 is now using them from `progs_cnp_vars_mod.F90`.

If you reply quickly, I can remove them and then approve the PR.

Reply:
Hmmm -they should be gone from here. I have done it somewhere. Perhaps in the miscellaneous science PR. I'll rm in this PR anyway. 91132f1